controllers/v2/model_controller_v2.py

```diff
@@ -31,14 +31,34 @@ def __init__(
         self.data_service = data_service
 
     def get_models_by_type(self, model_type: str) -> List[QAModelResponseV2]:
-        models: List[QAModel] = self.service.read_all_qa_model_by_type(model_type)
+        models: List[QAModel] = self.service.read_all_qa_model_by_type(
```

Comment:
Any reason for the new line in this function call?  Seems like a one liner would make more sense.

Reply:
Auto formatter probably, first time messing with it.